build.zig

```diff
@@ -648,6 +660,17 @@ fn java_client(
     bindings.setMainPkgPath("src");
     const bindings_step = bindings.run();
 
+    // We might need to clone JUI, if it doesn't exist
+    if (!file_or_directory_exists("src/clients/java/lib/jui") and !git_clone_jui) {
+        const git_clone = b.addSystemCommand(&.{ "git", "clone", "https://github.com/zig-java/jui.git", "src/clients/java/lib/jui" });
```

Comment:
It feels so weird to inform the commit hash here, but at the same time, it's the safer choice for reproducible builds.

For now, we can also check out the branch "zig-0.9.1", that dependency will be removed soon anyway.
```suggestion
        const git_clone = b.addSystemCommand(&.{ "git", "clone", "-b", "zig-0.9.1", "https://github.com/zig-java/jui.git", "src/clients/java/lib/jui" });
```